.github/workflows/node.working.js.yml

```diff
@@ -0,0 +1,186 @@
+name: OHRI CI - @esm and Docker Image
+
+on:
+  push:
+    branches:
+      - working
+
+  pull_request:
+    branches:
+      - working
+    types: [opened, synchronize]
+  release:
+    types:
+      - created
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+
+      - name: Cache dependencies
+        id: cache
+        uses: actions/cache@v3
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-${{ hashFiles('**/yarn.lock') }}
+
+      - name: Install dependencies
+        if: steps.cache.outputs.cache-hit != 'true'
+        run: yarn install --immutable
+
+      - name: Run tests
+        run: yarn run test
+
+      # Temporarily disable typecheck
+      # - name: Run lint and typecheck
+      #   run: yarn turbo run lint typescript --color --api="http://127.0.0.1:9080" --token="${{ secrets.TURBO_SERVER_TOKEN }}" --team="${{ github.repository_owner }}"
+
+      - name: Run build
+        run: yarn turbo run build --color --concurrency=5 --api="http://127.0.0.1:9080" --token="${{ secrets.TURBO_SERVER_TOKEN }}" --team="${{ github.repository_owner }}"
+
+      - name: Upload build artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: packages
+          path: |
+            packages/**/dist
+
+  pre_release:
```

Comment:
pre_release has always run on dev. Is this going to change?

Reply:
dev and working are similar and run on `next`

Reply:
But this would mean a pre-release is only run after code gets into working which will delay the workflow. Also adding this to working and dev means the same code will be published twice

Reply:
Resolved. ohri-working needs to create frontend docker images and deploy. 'next' version published already by dev ci @pirupius